recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "tiledbsoma" %}
-{% set version = "1.3.0" %}
-{% set version_r = "0.99.1" %}
-{% set sha256 = "1fb5bf86440a47b724ffb083143bda8a26d79e036336bac57ac7c1feaa5d4b9d" %}
+{% set version = "1.4.0" %}
+{% set version_r = "1.4.0" %}
```

Comment:
Shall we replace the `version_r` and just use `version` now?

Reply:
yes; good call definitely